### PR TITLE
Fedora 26.

### PR DESCRIPTION
**Pull request by adelton**

### CODE REVIEW

File: Dockerfile.fedora-26 [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Clone from the Fedora 26 image
+FROM registry.fedoraproject.org/fedora:26
+
+MAINTAINER FreeIPA Developers <freeipa-devel@redhat.com>
```
Comment (MartinBasti) [inline Dockerfile.fedora-26]: Mailing list was changed `freeipa-devel@lists.fedorahosted.org`

File: Dockerfile.fedora-26 [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+RUN groupadd -g 288 kdcproxy ; useradd -u 288 -g 288 -c 'IPA KDC Proxy User' -d '/var/lib/kdcproxy' -s '/sbin/nologin' kdcproxy
+
+COPY atomic-install-help /usr/share/ipa/
+COPY volume-data-list volume-data-mv-list volume-data-autoupdate /etc/
+RUN set -e ; cd / ; mkdir /data-template ; cat /etc/volume-data-list | while read i ; do echo $i ; if [ -e $i ] ; then tar cf - .$i | ( cd /data-template && tar xf - ) ; else mkdir -p /data-template$( dirname $i ) ; fi ; mkdir -p $( dirname $i ) ; if [ "$i" == /var/log/ ] ; then mv /var/log /var/log-removed ; else rm -rf $i ; fi ; ln -sf /data${i%/} ${i%/} ; done
+RUN rm -rf /var/log-removed
+RUN sed -i 's!^d /var/log.*!L /var/log - - - - /data/var/log!' /usr/lib/tmpfiles.d/var.conf
+# Workaround 1286602
+RUN mv /usr/lib/tmpfiles.d/journal-nocow.conf /usr/lib/tmpfiles.d/journal-nocow.conf.disabled
+RUN mv /data-template/etc/dirsrv/schema /usr/share/dirsrv/schema && ln -s /usr/share/dirsrv/schema /data-template/etc/dirsrv/schema
```
Comment (martbab) [inline Dockerfile.fedora-26]: This workaround for dirsrv should not be necessary, in fact it breaks the subsequent deployment of the image:

```console
2017-05-24T16:09:28Z DEBUG Process finished, return code=1
2017-05-24T16:09:28Z DEBUG stdout=[17/05/24:16:09:28] - [Setup] Info Could not copy file '/etc/dirsrv/schema/schema' to '/etc/dirsrv/slapd-EXAMPLE-TEST/schema/schema'.  Error: Is a directory
Could not copy file '/etc/dirsrv/schema/schema' to '/etc/dirsrv/slapd-EXAMPLE-TEST/schema/schema'.  Error: Is a directory
```

File: Dockerfile.fedora-26 (hunk 1 above)
Comment (adelton) [inline Dockerfile.fedora-26]: Fixed in 00894a54dedfa401577d12507fd25215934051e0.

File: Dockerfile.fedora-26 (hunk 2 above)
Comment (adelton) [inline Dockerfile.fedora-26]: Fixed in f0b868d1cedc98756fab1c20caa877119d608140.

**Comment by martbab** [comment]
LGTM, except of some AVCs I observe when testing the image:

```console
time->Tue May 30 06:33:34 2017
type=AVC msg=audit(1496140414.431:540): avc:  denied  { module_request } for  pid=30961 comm="ntpd" kmod="net-pf-0" scontext=system_u:system_r:container_t:s0:c30,c644 tcontext=system_u:system_r:kernel_t:s0 tclass=system permissive=0
----
time->Tue May 30 06:33:35 2017
type=AVC msg=audit(1496140415.417:542): avc:  denied  { module_request } for  pid=30961 comm="ntpd" kmod="net-pf-0" scontext=system_u:system_r:container_t:s0:c30,c644 tcontext=system_u:system_r:kernel_t:s0 tclass=system permissive=0
----
time->Tue May 30 06:33:35 2017
type=AVC msg=audit(1496140415.418:543): avc:  denied  { module_request } for  pid=30961 comm="ntpd" kmod="net-pf-0" scontext=system_u:system_r:container_t:s0:c30,c644 tcontext=system_u:system_r:kernel_t:s0 tclass=system permissive=0
----
time->Tue May 30 06:33:38 2017
type=AVC msg=audit(1496140418.418:548): avc:  denied  { module_request } for  pid=30961 comm="ntpd" kmod="net-pf-0" scontext=system_u:system_r:container_t:s0:c30,c644 tcontext=system_u:system_r:kernel_t:s0 tclass=system permissive=0
----
time->Tue May 30 06:33:36 2017
type=AVC msg=audit(1496140416.417:544): avc:  denied  { module_request } for  pid=30961 comm="ntpd" kmod="net-pf-0" scontext=system_u:system_r:container_t:s0:c30,c644 tcontext=system_u:system_r:kernel_t:s0 tclass=system permissive=0
----
time->Tue May 30 06:33:36 2017
type=AVC msg=audit(1496140416.418:545): avc:  denied  { module_request } for  pid=30961 comm="ntpd" kmod="net-pf-0" scontext=system_u:system_r:container_t:s0:c30,c644 tcontext=system_u:system_r:kernel_t:s0 tclass=system permissive=0
----
time->Tue May 30 06:33:36 2017
type=AVC msg=audit(1496140416.418:546): avc:  denied  { module_request } for  pid=30961 comm="ntpd" kmod="net-pf-0" scontext=system_u:system_r:container_t:s0:c30,c644 tcontext=system_u:system_r:kernel_t:s0 tclass=system permissive=0
----
time->Tue May 30 06:33:38 2017
type=AVC msg=audit(1496140418.418:547): avc:  denied  { module_request } for  pid=30961 comm="ntpd" kmod="net-pf-0" scontext=system_u:system_r:container_t:s0:c30,c644 tcontext=system_u:system_r:kernel_t:s0 tclass=system permissive=0
Fail: AVC messages found.
Checking for errors...
```

I shall open a BZ against docker in F26 to address these but otherwise it seems that F26 image works so I will merge this PR.

**Comment by adelton** [comment]
> I shall open a BZ against docker in F26 to address these

Right, ideally with some minimal reproducer of just `ntpd`.

**Pull request merged**